langchain/document_loaders/rtf.py

```diff
@@ -0,0 +1,22 @@
+"""Loader that loads rich text files."""
+from typing import List
+
+from langchain.document_loaders.unstructured import (
+    UnstructuredFileLoader,
+    satisfies_min_unstructured_version,
+)
+
+
+class UnstructuredRTFLoader(UnstructuredFileLoader):
+    """Loader that uses unstructured to load rtf files."""
+
+    def _get_elements(self) -> List:
+        min_unstructured_version = "0.5.12"
+        if not satisfies_min_unstructured_version(min_unstructured_version):
```

Comment:
Could we do this in the `__init__` please?

Reply:
Sure thing! Will move it to `__init__`